CONTRIBUTING.md

```diff
@@ -29,17 +29,29 @@ Declaring formal releases remains the prerogative of the project maintainers.
 
 This is an experiment and feedback is welcome! This document may also be subject to pull-requests or changes by contributors where you believe you have something valuable to add or change.
 
-## Onboarding Collaborators
+# Fastify Organization Structure
 
-> This section pertains to Fastify collaborators
+We have many collaborators, and every kind of contribution is welcome! Our organization is structured as follows:
+
+|  Team | Competence  |  Repository |
+|---|---|---|
+| `@fastify/core`   |  Fastify Core development  |  `fastify`, `fast-json-stringify`, `light-my-request`, `fastify-plugin`, `middie` |
+| `@fastify/plugins`   |  Build, maintain and release Fastify plugins  |  All plugins repositories |
+| `@fastify/benchmarks`   |  Build and maintain our benchmarks suite  |  `benchmarks` |
+| `@fastify/docs-chinese`   |  Translate the Fastify documentation in Chinese  |  `docs-chinese` |
+
+Every memeber of the org is also part of `@fastify/fastify`.
```

Comment:
We should also document here the relationship to npm permissions, e.g. `fastify` and its direct dependency are only released by the lead maintainers.

Reply:
Are "lead maintainers" the "core" team? This should be defined.